impl/src/main/resources/org/jboss/cdi/tck/tests/full/alternative/broken/incorrect/name/beans.xml

```diff
@@ -4,6 +4,6 @@
          https://jakarta.ee/xml/ns/jakartaee/beans_3_0.xsd"
        version="3.0" bean-discovery-mode="all">
     <alternatives>
-        <class>org.jboss.cdi.tck.tests.alternative.broken.incorrect.name.NonExistingClass</class>
+        <class>org.jboss.cdi.tck.tests.full.alternative.broken.incorrect.name.NonExistingClass</class>
```

Comment:
Shouldn't the other `beans.xml` files that have been moved also have a change like this?

Reply:
@Ladicek not sure I get you, there are 5 beans.xml files moved like this.
The other beans.xml files were either completely empty (for instance it has `<interceptors>` but not listing any) or were referencing non-existent classes (as intended part of the test). Those classes might have had different packages altogether so IDEA didn't change them when I issued a package change :)

Reply:
Fair enough, I was just curious :-)